Spanner/tests/System/TransactionTest.php

```diff
@@ -264,6 +265,112 @@ public function testRunTransactionWithDbRole($db, $values, $expected)
         }
     }
 
+    // @TODO: Need to work on this once the directedRead is available
+    /**
+     * @dataProvider getDirectedReadOptions
```

Comment:
Using `getDirectedReadOptions` can you also add a case when the directed read options yields a successful result?

Reply:
Done